<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>215</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>제품 종류</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="lb_no">
         <property name="text">
          <string>번호</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_no"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="lb_category">
         <property name="text">
          <string>종류</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_category"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btn_ins_category">
       <property name="text">
        <string>추가 (&amp;i)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_edit_category">
       <property name="text">
        <string>편집 (&amp;e)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_del_category">
       <property name="text">
        <string>삭제 (&amp;d)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QTableWidget" name="table_category"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
